### PR TITLE
adding check if title_text exists and is a string

### PR DESCRIPTION
**Pull request by ian28223**
adding check if title_text exists and is a string else use the query as graph title

### CODE REVIEW

**Comment by CharlyF** [comment]
Okay, this looks good code wise.
Could you just add a comment to document and explain this change ?
Thanks for the formatting changes as well.

**Comment by ian28223** [comment]
Needs title_check outside of the current if/else clause to affect hostmap type widgets

File: dashconverter.py [hunk 1]
```diff
@@ -89,21 +89,23 @@ def widget_transform(cls):
...
 				if 'conditional_formats' not in widgets[i]['tile_def']['requests'][0]:
 					widgets[i]['tile_def']['requests'][0]['conditional_formats'] = []
-			else: 
+			else:
 				widgets[i]['tile_def'] = 'outdated'
 				print "One of the widgets' type is outdated and won't be ported.\n To solve this, just click on edit the dashboard, open a widget, hit done and save the dashboard.\n Then run the script again."
-				print widgets[i]
 
-			if widgets[i]['type'] == 'hostmap':
+			if not (('title_text' in widgets[i]) and (isinstance(widgets[i]['title_text'],str))):
```
Comment (ian28223) [inline dashconverter.py]: Checks if the title_text property exists in widgets[i] and checks if the value is a valid string if it does exists.
If not, create/set the widget's title_text property to the query e.g. title_text = "avg:system.cpu.user{*}"

**Comment by CharlyF** [comment]
👍

**Pull request merged**